pages.fr/common/mongo.md

```diff
@@ -0,0 +1,20 @@
+# mongo
+
+> Client shell pour MongoDB
+> Plus d'informations: <https://docs.mongodb.com/manual/reference/program/mongo>.
+
+- Se connecter à une base de données (database):
```

Comment:
I don't speak the language (but sounds related to Spanish) but we use imperative mood here. I think this should be:
```suggestion
- Se connecte à une base de données (database):
```

Reply:
You are right, in french, imperative of the verb `connecter` is whether:
- `connecte` (when you are talking to one person)
- `connectez` (when you are talking to more than one persons (excluding yourself) )
- `connectons` (when you are talking about you + one or more persons)

in the future i will be more careful on this :D

Reply:
I have a look to other files translated in french (in order be inspired and have the same syntax)

According to this translation [base32](https://github.com/tldrpages/tldr/blob/main/pages.fr/common/base32.md) (and many other) They do not use demonstrative pronouns like `Se` at beginning of each impérative verbs. 

I'll remove them :D 

PS: i noticed the translation [base64](https://github.com/tldrpages/tldr/blob/main/pages.fr/common/base64.md) is not in imperative. Should i create an issue and fix it ? or is it ok ?

Reply:
@Nico385412 yeah just send the PR with the fix.

Reply:
Done :D 
https://github.com/tldr-pages/tldr/pull/6706